OpenOversight/scripts/entrypoint_dev.sh

```diff
@@ -0,0 +1,3 @@
+yarn build
```

Comment:
I'm not sure if it's absolutely necessary, but since you have it in the prod version, you should add `#!/bin/bash` to the top of this file. 

It's also be good to add comments on what it's doing like you did in the prod version.

Reply:
+1 to this!

Reply:
Good points, thanks! I added `#!/bin/bash` and made executable. Also added comments.

Reply:
Noice!